akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala

```diff
@@ -412,6 +412,20 @@ object Source {
     })
 
   /**
+   * Combine the elements of multiple streams into a stream of sequences.
+   */
+  def zipN[T](sources: Seq[Source[T, _]]) = zipWithN((s: Seq[T]) ⇒ s)(sources)
```

Comment:
Oh! I don't know why, but I was pretty sure that the default `Seq` was the immutable one.